.editorconfig

```diff
@@ -7,15 +7,10 @@ indent_size = 2
 trim_trailing_whitespace = true
 
 [*.{gradle,gradle.kts}]
-charset = utf-8
-indent_style = space
 indent_size = 4
 insert_final_newline = true
 max_line_length = 120
 
 [*.{java,groovy}]
-charset = utf-8
-indent_style = space
-indent_size = 2
```

Comment:
Why remove these?

Reply:
They are defined above with the same values. Just OCD things.